builder.go

```diff
@@ -37,6 +37,8 @@ type Copy struct {
 type Run struct {
 	Shell bool
 	Args  []string
+	// Mounts are mounts specified through the --mount flag inside the dockerfile
```

Comment:
Only if you've other changes
```suggestion
	// Mounts are mounts specified through the --mount flag inside the Dockerfile
```

Reply:
Better yet call it Containerfile

Reply:
imagebuilder doesn't look for files with that name.